pkgs/applications/misc/flashprint/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, libGLU, qtbase, fetchurl, dpkg, autoPatchelfHook, wrapQtAppsHook }:
+
+let
+  pname = "flashprint";
+  version = "5.7.1";
+  sha256 = "sha256-kxvqEgXlKQlfzlCqKb5o3hvop82vDsJmQDK9XOCq61g=";
+  url = "http://www.ishare3d.com/3dapp/public/FlashPrint-5/FlashPrint/flashprint5_${version}_amd64.deb";
+
+in stdenv.mkDerivation {
+  inherit pname version;
+  src = fetchurl { inherit url sha256; };
+  nativeBuildInputs = [ dpkg autoPatchelfHook wrapQtAppsHook ];
+  buildInputs = [ qtbase libGLU ];
+  unpackPhase = ''
+    dpkg-deb -x $src .
+  '';
```

Comment:
Is this really necessary? `dpkg` has a setup hook to do this automatically.

Reply:
Removed unpackPhase, thanks!